libs/ml/include/ml/ops/weights.hpp

```diff
@@ -89,12 +93,26 @@ class Weights : public fetch::ml::ops::PlaceHolder<T>, public Trainable<T>
     return false;
   }
 
-  virtual void Step(typename T::Type learningRate)
+  virtual void Step(typename T::Type learning_rate)
   {
-    this->gradient_accumulation_->InlineMultiply(-learningRate);
+    this->gradient_accumulation_->InlineMultiply(-learning_rate);
     this->output_->InlineAdd(*gradient_accumulation_);
     // Major DL framework do not do that, but as I can't think of any reason why, I'll leave it here
```

Comment:
remove comment

Reply:
Comment removed, but I will remove Step as part of refactorization anyway.